src/explore-education-statistics-common/src/modules/table-tool/components/__tests__/DataTableCaption.test.tsx

```diff
@@ -0,0 +1,193 @@
+import {
+  CategoryFilter,
+  Indicator,
+  LocationFilter,
+  TimePeriodFilter,
+} from '@common/modules/table-tool/types/filters';
+import { FullTableMeta } from '@common/modules/table-tool/types/fullTable';
+import { generateTableTitle } from '../DataTableCaption';
+
+describe('DataTableCaption', () => {
+  const generateTableTitleBasicInput: FullTableMeta = {
+    geoJsonAvailable: false,
+    publicationName: 'Pupil absence in schools in England',
+    subjectName: 'Absence by characteristic',
+    footnotes: [],
+    boundaryLevels: [],
+    filters: {
+      Characteristic: {
+        name: 'characteristic',
+        options: [
+          new CategoryFilter({
+            value: 'total',
+            label: 'Total',
+            group: 'Gender',
+            category: 'Characteristic',
+          }),
+        ],
+      },
+    },
+    indicators: [
+      new Indicator({
+        label: 'Authorised absence rate',
+        value: 'authAbsRate',
+        unit: '%',
+        name: 'sess_authorised_percent',
+      }),
+    ],
+    locations: [
+      new LocationFilter({
+        value: 'E92000001',
+        label: 'England',
+        level: 'country',
+      }),
+    ],
+    timePeriodRange: [
+      new TimePeriodFilter({
+        code: 'AY',
+        year: 2015,
+        label: '2015/16',
+        order: 0,
+      }),
+    ],
+  };
+
+  describe('generateTableTitle', () => {
```

Comment:
Refactor these tests to actually test what is rendered by the caption. There's no point in just testing this function as it's not comprehensive enough.